nixos/modules/services/monitoring/prometheus/exporters/deluge.nix

```
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, options, ... }:
```

Doesn't seem to be used

```suggestion
{ config, lib, pkgs, ... }:
```